resources/views/vue.blade.php

```diff
@@ -179,8 +179,42 @@ class="spaced"
                 ></v-chip>
             </template>
 
+            <!-- Route Path -->
             <template v-slot:item.path="{ item }">
-                <span v-html="highlightParameters(item.path)"></span>
+                {{-- Open Link in New Tab --}}
+                @if (config('pretty-routes.show_path_link', false))
+                    <a v-bind:href="'{{url('/')}}/' + getDummyPath(item.path)" target="_blank" class="text-decoration-none">
+                        <v-icon dense="true" style="margin-right: 5px">mdi-open-in-new</v-icon>
+                    </a>
+
+                    <span v-html="highlightParameters(item.path)"
+                    @if (config('pretty-routes.double_click_and_copy', false))
```

Comment:
You should move the output of the variable from php to a javascript variable.

This is now creating duplicate code.